.github/workflows/ci.yml

```diff
@@ -273,19 +265,11 @@ jobs:
         if: ${{ runner.os == 'macOS' || runner.os == 'Linux' }}
         run: |
           echo "127.0.0.1   sentry.native.test" | sudo tee -a /etc/hosts
+          # remove "::1 localhost ..." to avoid conflicts with proxy tests (musl)
+          sed '/^::1/d' /etc/hosts | sudo tee /etc/hosts
```

Comment:
`/etc/hosts` is a special bind-mounted file in GitHub Actions. I tried doing this in a Docker ENTRYPOINT script, but it was too early because the changes were lost at this point...